torch/testing/_internal/common_methods_invocations.py

```diff
@@ -976,81 +973,25 @@ def __init__(self,
         self.ref = ref
 
 
-class HermitianOpInfo(OpInfo):
```

Comment:
I'm not sure if this PR can remove this class. cc @IvanYashchuk, you're updating this class in another PR of yours, right?

Reply:
It's great that `grad_func=` replaces `HermitianOpInfo` and `TriangularOpInfo`! But it will hit a merge conflict with https://github.com/pytorch/pytorch/pull/51107
Let's wait until that PR is merged and then rebase this one.